### PR TITLE
Feature/render tweets

### PR DESCRIPTION
**Pull request by jungeunyoon**
**UI**
@Rise-Vision/creative  You should be able to check the styling with `rise-twitter` component alone with`npm install` `npm run build` and `npm run start`. Go to `localhost:3000` and you should see the filler data pulled from the company Twitter account.

### CODE REVIEW

**Comment by jungeunyoon** [comment]
[Twitter Component renders [2]](https://trello.com/c/OpzTc6SL/4647-twitter-component-renders-2)

**Comment by sheadarlison** [comment]
@jungeunyoon I'm sorry for the switch up but could we use the font that was used in the original designs @mmeiers did and the color. I think it's an off black and the font is Lato. @mmeiers can you let Jaime know the exact colors and font weights, etc... Also could you make the default layout the light text on dark, I think it will be a better default because I was just going through our dribble and noticed most designs we do are light text on a dark background. 

I also put together a schedule here for testing all of the different ratios at different sizes. 

Schedule: https://apps.risevision.com/schedules/details/4c59d7f7-f48b-4be5-b125-a409c3c9fd6c?cid=7fa5ee92-7deb-450b-a8d5-e5ed648c575f

ScreenConnect (if you're curious): https://connect.risevision.com/Host#Access/*Rise%20Display%20Creative//369432da-838f-4541-9c11-7c6fbd83d3d0 

Jaime, you might want to add your display to the templates account and add it to this schedule so you can watch it at home. 

Also it looks like when the 25 tweets are done fading through they don't restart. 

Here's a link to a google drive folder so you can see what it looks like on my display.

https://drive.google.com/open?id=108K8HupAGX7QwcFOGd7cBTr4HYDhCp1S

Thanks!

Let me know if you have any questions.

**Comment by mmeiers** [comment]
FONT
The original font was 
Lato
<link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
font-family: 'Lato', sans-serif;

WEIGHT
Everything should be Regular (400) except for the name and date which should be bold (700).

COLOR
The off black color is #14171A. 
Lets use this color for all instances of black.

The retweet and heart icons/numbers should be this same black (#14171A) only at 50% opacity.
On the dark background layout it would just be the white (#FFF) at 50%.
(sorry i think i had 2 black colors when I changed the font in a hurry) 

Thanks,
Mat

**Comment by jungeunyoon** [comment]
@sheadarlison @mmeiers 
- it's not meant to repeat. this is strictly rendering. `Configure, Ready, Play, Pause, Stop and Done events.` is a separate card. 
- concerned about the load constraint of using lato. each component will have to load its own since their styles are encapsulated by their shadowdoms so it might take a toll. was this addressed in any way with previous widgets?
- font-sizes were calculated exactly so they should be correct. calculations could've been off, though, so please point out specific ones if you think they might be wrong. 
- will change font weights, though I'm pretty sure that they're correct as well.
- will change the colors colors 👍 
- yup i can definitely change the default theme to be `dark` since the styles already exist for it. FYI if you guys continue to test directly through component you won't be able to see anything since the body is white by default.

**Comment by sheadarlison** [comment]
@jungeunyoon 

1. Okay makes sense, wasn't sure on that. 
2. I'm not sure how the other widgets did it, @rodrigopavezi would know, I think if you're just loading the font at 400&700 it shouldn't cause performance issues, but I defer to Rodrigo or Stu.
3. Sounds good re. font sizes. They look correct just wanted to double check. 
4. Sounds good re. font weights. 
5. Sounds good re. colors.
6. Okay thanks.

**Comment by rodrigopavezi** [comment]
@jungeunyoon each widget instance is loaded inside an iframe so I think the old widgets load the same css multiple times. @stulees can you confirm? Cheers

**Comment by jungeunyoon** [comment]
@rodrigopavezi okay good to hear 👍 wouldn't want to break anything hahaha.

**Comment by stulees** [comment]
@jungeunyoon @rodrigopavezi Can you provide more detail on what the concern is, I'm not sure I follow, thanks.

File: src/rise-twitter.html [hunk 1]
```diff
@@ -1,8 +1,2 @@
-<style>
-  :host {
-    display: flex;
-  }
-</style>
-<div id="twitter-component-template">
-  <h1>Rise Twitter Component</h1>
-</div>
+<link rel="stylesheet" type="text/css" href="../dist/static/css/main.css">
```
Comment (rodrigopavezi) [inline src/rise-twitter.html]: @jungeunyoon I am not sure this href path will work on the deployed component in player. Shouldn't it be changed by webpack depending if it is a dev or prod build? cheers

File: src/tweet.js [hunk 2]
```diff
@@ -1,13 +1,224 @@
...
+  }
+
+  setTheme(settings) {
+    switch (settings.theme) {
+      case 'dark':
+        this.shadowRoot.querySelector('.twitter-component-template').removeClass('theme-light');
+        this.shadowRoot.querySelector('.twitter-component-template').addClass('theme-dark');
+        break;
+      default:
+        this.shadowRoot.querySelector('.twitter-component-template').removeClass('theme-dark');
```
Comment (rodrigopavezi) [inline src/tweet.js]: @jungeunyoon just a remainder to change default to dark. cheers

File: src/rise-twitter.html [hunk 3]
```diff
@@ -1,8 +1,3 @@
-<style>
-  :host {
-    display: flex;
-  }
-</style>
-<div id="twitter-component-template">
-  <h1>Rise Twitter Component</h1>
-</div>
+<link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Lato" media="all">
+<link rel="stylesheet" type="text/css" href="../dist/static/css/main.css">
```
Comment (rodrigopavezi) [inline src/rise-twitter.html]: @jungeunyoon curious, did you check if this will work on player because this hardcoded path to the main.css? Cheers

Comment (jungeunyoon) [reply src/rise-twitter.html]: @rodrigopavezi Yup, looking into it. 👍 There's quite a conflict in some new dependencies that had to be added to fix the issue.

**Pull request merged**